packages/platforms/accton/arm64/as4224/src/modules-5_6/arm64-accton-as4224-psu.c

```diff
@@ -166,6 +199,24 @@ static int as4224_psu_get_psu_count(void)
     return (status & 0x10) ? 1 : 2;
 }
 
+static int get_platform_id(void)
+{
+    int status;
+
+    status = as4224_psu_read_value(BOARD_INFO_REG_OFFSET);
+    if (status < 0)
+        return status;
+
+    if (status & 0x10)
```

Comment:
Is the datasheet/specification pubilc documenting these bits?

Reply:
Yes, it is described in the specification.